src/utils.js

```diff
@@ -34,11 +34,22 @@ function stringHasForbiddenCantPublishBecauseVersionExists(str) {
 }
 
 function getPackageVersionInfo(registry, packageName, version) {
+  const npmPackParams = `show --json --registry=${registry} ${packageName}@${version}`;
   return JSON.parse(
-    execSync(`npm show --json --registry=${registry} ${packageName}@${version}`, { stdio: 'pipe' }).toString(),
+    execa.sync('npm', npmPackParams.split(' ')),
```

Comment:
the cost for async is low here.

also, if the packageName has lot's of versions, it will block wix-fed-scripts from doing other stuff.

---

im only pointing this up because you make this code prettier ;D